qiskit_sphinx_theme/__init__.py

```diff
@@ -25,6 +27,35 @@ def _get_theme_absolute_path(folder_name: str) -> str:
     return str(path.resolve())
 
 
+def remove_thebe_if_not_needed(app, pagename, templatename, context, doctree):
+    """
+    Remove files that jupyter-sphinx incorrectly tries to add.
+    
+    See https://github.com/Qiskit/qiskit_sphinx_theme/issues/291 for more context. 
+    """
+    # jupyter-sphinx might be not installed. 
+    try:
+        from jupyter_sphinx.thebelab import ThebeButtonNode
+    except ImportError:
+        return 
+
+    if not doctree:
+        return
+    
+    if not doctree.traverse(ThebeButtonNode):
+        new_script_files = []
+        for js_file in context["script_files"]:
+            if js_file not in ["_static/sphinx-thebe.js", "_static/thebelab-helper.js", "https://unpkg.com/thebelab@latest/lib/index.js"]:
+                new_script_files.append(js_file)
+        context["script_files"] = new_script_files
```

Comment:
This can be more concise by using a list comprehension. List comprehensions are seen as more idiomatic than building a list via a `for loop`, in part because it makes it more clear what you're doing: building a new list.

```suggestion
        thebe_js_files = [
            "_static/sphinx-thebe.js",
            "_static/thebelab-helper.js",
            "https://unpkg.com/thebelab@latest/lib/index.js",
        ]
        context["script_files"] = [
            js_file for js_file in context["script_files"] if js_file not in thebe_js_files
        ]
```

Please make a similar change for `context["css_files"]`.